### ISSUE TITLE
Times and Dates

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#571]
Issue group: Time and dates
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: 

### Description
Times and dates should be in a <time> element with the datetime attribute set to a machine readable format. This allows ATs to properly interpret and present the time/date to the user.

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-time-element

![]()

**Comment by adriannelee** [comment]
We don't where this issue lives, we would need to know to solve the problem. Time is an important aspect of Notify, especially when it comes to limits

**Comment by YedidaZalik** [comment]
Phil can work on this

**Comment by YedidaZalik** [comment]
Got reviews from Andrew
Can move to done
